apps/api/src/domain/models/audio-item/commands/transcripts/add-participant-to-transcript/add-participant-to-transcript.command.ts

```diff
@@ -0,0 +1,31 @@
+import { AggregateCompositeIdentifier, ICommandBase } from '@coscrad/api-interfaces';
+import { Command } from '@coscrad/commands';
+import { NestedDataType, NonEmptyString } from '@coscrad/data-types';
+import { AudioItemCompositeIdentifier } from '../../../entities/audio-item-composite-identifier';
+import { ADD_PARTICIPANT_TO_TRANSCRIPT } from '../constants';
+
+@Command({
+    type: ADD_PARTICIPANT_TO_TRANSCRIPT,
+    description: 'Adds a participant to a transcript',
+    label: 'Add Participant to Transcript',
+})
+export class AddParticipantToTranscript implements ICommandBase {
+    @NestedDataType(AudioItemCompositeIdentifier, {
+        label: 'Composite Identifier',
+        description: 'system-wide unique identifier',
+    })
+    readonly aggregateCompositeIdentifier: AggregateCompositeIdentifier;
+
+    @NonEmptyString({
+        label: `participant's initials`,
+        description: `the initials or identifier to use for this participant`,
+    })
+    readonly initials: string;
```

Comment:
like the initials!

Reply:
These serve as a locally unique ID. Within the context of a single audio item (aggregate composite id with type: audioItem and a given id: uuid), there is one and only one participant with the given initials. We prevent adding a second. However, there may be a participant with the same iniitals on the transcript for some other audio item.

The transcript is thus an entity whose state is controlled by the audio item aggregate root.